### PR TITLE
added jar file and readme

### PR DESCRIPTION
**Pull request by RachelRebecca**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+of the spell to get credit for the question.
+
+This project uses:
+- [Wizard World API](https://wizard-world-api.herokuapp.com/swagger/index.html)
+- [Retrofit](https://square.github.io/retrofit/)
+- Gson
+- [Mockito](https://site.mockito.org/)
+- Junit
+- Rx
+- Gradle
```
Comment (andrewoid) [inline README.md]: Add links for the other things in this list.

**Comment by andrewoid** [comment]
The rotating color for the questions can make them hard to read.

**Review by andrewoid — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (RachelRebecca) [reply README.md]: I hope I added accurate links

File: src/main/java/SpellColor.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+        spellColor = spellLightColors.lightToColor.get(spellColorName);
+        if (spellColor != null)
+        {
+            graphics.setColor(spellColor);
+            graphics.fillOval(0, 20, 15, 15);
+        } else
+        {
+            graphics.setColor(Color.BLACK);
+            graphics.drawOval(0, 20, 15, 15);
```
Comment (andrewoid) [inline src/main/java/SpellColor.java:35]: This line is the same as line 31 and can be "factored out" by putting it out of the if-else statement.

File: src/main/java/NewtsPresenter.java [hunk 3]
```diff
@@ -52,7 +50,13 @@ public void loadSpellInformation(String category)
         } else
         {
             view.setCategorySelected("");
-            getNewQuestion(category);
+            StringBuilder newCategory = new StringBuilder();
+            String[] splitCategory = category.split(" ");
+            for (String cat : splitCategory)
+            {
+                newCategory.append(cat);
+            }
```
Comment (andrewoid) [inline src/main/java/NewtsPresenter.java]: https://www.journaldev.com/18361/java-remove-character-string

**Review by andrewoid — APPROVED** [review]

**Pull request merged**